src/js/components/MultiSelect/propTypes.js

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import { genericProps } from '../../utils/general-prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    ...genericProps,
+    children: PropTypes.func,
+    defaultValue: PropTypes.arrayOf(
+      PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.object,
+        PropTypes.number,
+      ]),
+    ),
+    disabled: PropTypes.oneOfType([
```

Comment:
can we pull out the identical propTypes used by both select and mutli-select and then just import them so we only have to go to one place to change for both?

Reply:
done